.travis.yml

```diff
@@ -20,10 +18,14 @@ php:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: hhvm
```

Comment:
The ```HHVM``` is failed because it doesn't support the ```ReflectionClass```.
I use this feature because I want to test some methods directly.